webpack/assets/javascripts/foreman_toast_notifications.js

```diff
@@ -0,0 +1,46 @@
+import * as ToastActions from './react_app/redux/actions/toasts';
+import store from '../javascripts/react_app/redux';
+import $ from 'jquery';
+
+export function notify(message, type, sticky) {
```

Comment:
The Toast component is decoupled from redux. The ToastList ,the container component is not. This probably means a different container for implementations not using redux or requiring redux when using toasts.